EosPayload/drivers/led_driver.py

```diff
@@ -38,7 +37,12 @@ def device_command(self, logger: logging.Logger) -> None:
             self.thread_sleep(logger, 2)
 
     def cleanup(self):
-        super(LEDDriver, self).cleanup()
         GPIO.output(self.led_1, 0)
         GPIO.output(self.led_2, 0)
         GPIO.output(self.led_3, 0)
+
+        GPIO.cleanup(self.led_1)
+        GPIO.cleanup(self.led_2)
+        GPIO.cleanup(self.led_3)
```

Comment:
did you look up the function?  b/c if you did you'd see it takes no parameters and only needs to be called once
https://adafruit-bbio.readthedocs.io/en/latest/GPIO.html